### DISCUSSION TITLE
Mod-ui and MOD Duo firmware versions

### DISCUSSION THREAD

**Discussion by sonicdebris** [mod-audio/mod-ui#143] [Q&A] [answered]
Is there a way to understand which revision of the code in this repo corresponds to the one deployed in a certain mod duo firmware version? for example, which one should I check out to match firmware version 1.6.3?

**Comment by falkTX** [comment, marked as answer]
Just use the hotfix-* branches.
the v1.6.3 and v1.7.3 use hotfix-1.6 and hotfix-1.7 branches, respectively.

**Comment by sonicdebris** [comment]
Ah, thanks a lot!
